moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp

```diff
@@ -1184,9 +1184,15 @@ void MotionPlanningDisplay::onRobotModelLoaded()
 
   dynamics_solver_.clear();
   for (std::size_t i = 0; i < groups.size(); ++i)
-    if (getRobotModel()->getJointModelGroup(groups[i])->isChain())
-      dynamics_solver_[groups[i]].reset(
-          new dynamics_solver::DynamicsSolver(getRobotModel(), groups[i], gravity_vector));
+  {
+    const robot_model::JointModelGroup* jmg = getRobotModel()->getJointModelGroup(groups[i]);
+    if (jmg != nullptr)
+    {
```

Comment:
Is this parenthesis required?

Reply:
Required for the same reason as the comment on line 1187.

Reply:
OK.